request_information.go

```diff
@@ -468,6 +479,11 @@ func (request *RequestInformation) AddQueryParameters(source interface{}) {
 		arr, ok := value.([]string)

```

Comment:
I've added support for `[]any`. I don't think there's a good way to fill the old `QueryParameters` for this, because there's no real way to convert any type to a string. I could have used `fmt.Sprintf("%v", val)`, but that might give strange results for complex types. `[]any` wasn't supported anyway so backward compatibility is not that big of an issue here.